integration_tests/postgres/cleanup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set +e
+
+echo "postgres/cleanup: Begin tests cleanup for postgres"
+
+function clean() {
+  CONTAINER_NAME=$1
+  docker stop $CONTAINER_NAME
+}
+
+versions="9.3 9.4 9.5 9.6 10.1"
+types="ssl nossl"
+for version in $versions; do
+  for type in $types; do
+    clean "zgrab_postgres_${version}-${type}"
```

Comment:
Do we need to `docker rm` these as well?

Reply:
In [setup.sh](https://github.com/zmap/zgrab2/blob/feature/addPostgresModule/integration_tests/postgres/setup.sh), they are launched with the `--rm` flag, so they are automatically removed when they are stopped.